src/main/java/com/sarality/form/FormData.java

```diff
@@ -111,6 +111,16 @@ public void setDate(FormField field, DateTime value) {
     }
   }
 
+
+
+  public void setBoolean(FormField field, Boolean value) {
+    addValue(field, value.toString());
+  }
+
+  public Boolean getBoolean(FormField field)  {
+    return getValue(field) != null;
```

Comment:
What if that value has been set to FALSE? In that case this will return TRUE when the value is false